cuml/umap_demo_graphed.ipynb

```diff
@@ -325,5 +325,5 @@
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
```

Comment:
The graph is not showing up on the saved version, maybe it just needs to be rerun?

Reply:
No umap changes should be included in this PR. I’ll revert these .